sqlx-core/src/postgres/types/mod.rs

```diff
@@ -32,6 +33,10 @@ impl PgTypeInfo {
     pub fn with_oid(oid: u32) -> Self {
         Self { id: TypeId(oid) }
     }
+
+    pub fn oid(&self) -> u32 {
```

Comment:
We'd probably rather not expose this in the public API if possible (with custom types we may not always have an OID so this will need to be changed to `Option<u32>`). Either `pub(crate)` or `#[doc(hidden)]`.